.secrets.baseline

```diff
@@ -75,10 +75,6 @@
     {
       "path": "detect_secrets.filters.allowlist.is_line_allowlisted"
     },
-    {
-      "path": "detect_secrets.filters.common.is_baseline_file",
-      "filename": ".secrets.baseline"
-    },
```

Comment:
I've noticed that your PRs often remove this section from this file, and then other's put it back. Might be worth figuring out why and if it matters.